### PR TITLE
Manzil infinity180 patch 1 broken badge

### LINKED ISSUE #1901: [OTHER] Broken link of Images in ReadMe file
### What would you like to share?

Screenshot 2023-08-24 at 12 06 47 AM

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by manzil-infinity180**
closes #1901 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by schmelto — CHANGES_REQUESTED** [review]
image

this doesn't look quite right yet

**Review by manzil-infinity180 — COMMENTED** [review]
What i think i can change with shields.io i changed... I think now it look better than previous

**Comment by schmelto** [comment]
image

File: README.md [hunk 1]
```diff
@@ -7,20 +7,20 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
...
-    <img src="https://forthebadge.com/images/badges/built-with-love.svg" alt="built with love badge" />
+    <img src="https://img.shields.io/badge/built-with_love-blue" alt="built with love badge" />
  </a>
   <a href="https://github.com/EddieHubCommunity" target="_blank" rel="noopener noreferrer">
-    <img src="https://forthebadge.com/images/badges/made-with-markdown.svg" alt="made with markdown badge" />
+    <img src="https://img.shields.io/badge/made-with_love-pink" alt="made with markdown badge" />
  </a>
   <a href="https://github.com/EddieHubCommunity" target="_blank" rel="noopener noreferrer">
-    <img src="https://forthebadge.com/images/badges/open-source.svg" alt="open source badge" />
+    <img src="https://img.shields.io/badge/open-source-love" alt="open source badge" />
```
Comment (schmelto) [inline README.md:16]:
```suggestion
    <img src="https://img.shields.io/badge/open-source-green" alt="open source badge" />
```

File: README.md [hunk 2]
```diff
@@ -7,20 +7,20 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
...
  </a>
  <br />
  <a href="https://eddiehubcommunity.github.io/awesome-github-profiles/" target="_blank" rel="noopener noreferrer">
-    <img src="https://forthebadge.com/images/badges/check-it-out.svg" alt="check it out badge" />
+    <img src="https://img.shields.io/badge/check-it_out-love" alt="check it out badge" />
```
Comment (schmelto) [inline README.md:20]:
```suggestion
    <img src="https://img.shields.io/badge/check_it-out-blue" alt="check it out badge" />
```

File: README.md [hunk 3]
```diff
@@ -7,20 +7,20 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
...
  </a>
   <a href="https://github.com/EddieHubCommunity" target="_blank" rel="noopener noreferrer">
-    <img src="https://forthebadge.com/images/badges/built-by-developers.svg" alt="built by developers badge" />
+    <img src="https://img.shields.io/badge/built-by_developers-blue" alt="built by developers badge" />
```
Comment (schmelto) [inline README.md:23]:
```suggestion
    <img src="https://img.shields.io/badge/built_by-developers-yellow" alt="built by developers badge" />
```

**Comment by manzil-infinity180** [comment]
> image
> 
> Doesn't seem right they're should be like 'made with love' etc

Sir if you satisfied with the change Please merge it

**Review by schmelto — APPROVED** [review]

**Pull request merged**